faststart/cloud-in-a-box.sh

```diff
@@ -312,14 +312,14 @@ if [ "$?" == "0" ]; then
     exit 9
 fi
 
-# Check to see that we're running on CentOS or RHEL 6.5.
+# Check to see that we're running on CentOS or RHEL 6.6.
 echo "[Precheck] Checking OS"
-cat /etc/issue | egrep 'release.*6.[5-6]' 1>>$LOGFILE
+cat /etc/redhat-release | egrep 'release.*6.[5-6]' 1>>$LOGFILE
```

Comment:
Should we actually just be checking for 6.6 or should we change the error message below to say 6.5 or 6.6?